demisto_sdk/commands/validate/file_validator.py

```diff
@@ -540,7 +540,7 @@ def validate_added_files(self, added_files, received_file_type: str = None, modi
             print('\nValidating {}'.format(file_path))
             self.check_for_spaces_in_file_name(file_path)
 
-            if re.search(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE) and not file_type:
+            if re.search(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE) or not file_type:
```

Comment:
Actually, do we want to skip every file that we don't find it's file type?

Reply:
Why not just remove the file_type? why do we need it?

Reply:
you are right - made it work the same as in modified files